### PR TITLE
Meta: Fix isMilestone's test url

### PR DESCRIPTION
**Pull request by 134130**

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -149,7 +149,7 @@ addTests('isLabelList', [
 
 export const isMilestone = (url: URL | HTMLAnchorElement | Location = location): boolean => /^milestone\/\d+/.test(getRepo(url)?.path!);
 addTests('isMilestone', [
-	'https://github.com/sindresorhus/refined-github/milestone/12',
+	'https://github.com/sindresorhus/refined-github/milestones/12',
```
Comment (kang8) [inline index.ts:152]: `milestone` is right, you can see: https://github.com/kubernetes/kubernetes/milestone/56

Comment (134130) [reply index.ts:152]: @kang8 Oops, I thought issue page with milestone filter is milestones page.
I'll change this url as you suggenstioned. Thank you!

Comment (134130) [reply index.ts:152]: You can check the url on example is 404 page.
https://github.com/refined-github/refined-githu/milesone/12

**Review by kang8 — APPROVED** [review]
LGTM

File: index.ts (hunk 1 above)
Comment (134130) [reply index.ts]: Out of conversation, why GH is using resources path messy?
for example, issue's path is https://github.com/refined-github/refined-github/issues/6000 but pr is https://github.com/refined-github/refined-github/pulls/100

Comment (fregante) [reply index.ts]: > why GH is using resources path messy?

Consistency is hard on large projects 😃

**Comment by fregante** [comment]
Note that these URLs are not expected to exist so I'm sure there are dozens of such examples

**Comment by 134130** [comment]
Aha...
It was a bit of a hassle to find a sample page to verify that the feature works

**Comment by fregante** [comment]
> It was a bit of a hassle to find a sample page to verify that the feature works

Indeed, that's why we've been adding "Test URLs" at the bottom of each feature. For older features you might find the test URLs in their old PRs.

**Pull request merged**